kits/arms/ex_impedance_control.m

```diff
@@ -86,16 +88,16 @@
 %     gainsInEndEffectorFrame = true;
 %     damperGains = [0; 0; 0; .1; .1; .1;]; % (N/(m/sec)) or (Nm/(rad/sec))
 %     springGains = [0; 0; 0; 5; 5; 5];  % (N/m) or (Nm/rad)
- 
-    % HOLD POSITION AND ROTATION - BUT ALLOW MOTION ALONG/AROUND Z-AXIS
-    gainsInEndEffectorFrame = true;
-    damperGains = [10; 10; 0; .1; .1; .1;]; % (N/(m/sec)) or (Nm/(rad/sec))
-    springGains = [500; 500; 0; 5; 5; 5];  % (N/m) or (Nm/rad)
+  
+%     % HOLD POSITION AND ROTATION - BUT ALLOW MOTION ALONG/AROUND Z-AXIS
+%     gainsInEndEffectorFrame = true;
+%     damperGains = [10; 10; 0; .1; .1; .1;]; % (N/(m/sec)) or (Nm/(rad/sec))
+%     springGains = [500; 500; 0; 5; 5; 5];  % (N/m) or (Nm/rad)
     
-%     % HOLD POSITION AND ROTATION - BUT ALLOW MOTION IN BASE FRAME XY-PLANE
-%     gainsInEndEffectorFrame = false;
-%     damperGains = [0; 0; 5; .1; .1; .1;]; % (N/(m/sec)) or (Nm/(rad/sec))
-%     springGains = [0; 0; 500; 5; 5; 5];  % (N/m) or (Nm/rad)
+    % HOLD POSITION AND ROTATION - BUT ALLOW MOTION IN BASE FRAME XY-PLANE
+    gainsInEndEffectorFrame = false;
+    damperGains = [0; 0; 5; .1; .1; .1;]; % (N/(m/sec)) or (Nm/(rad/sec))
+    springGains = [0; 0; 500; 5; 5; 5];  % (N/m) or (Nm/rad)
 
```

Comment:
Why did you change the default mode? Do you think that's a better entry demo? I like hold-position with free orientation most